src/WinRT.Runtime/Interop/EventHandlerEventSource.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace WinRT.Interop
```

Comment:
These types maybe should be in the ABI namespace.  Thoughts?

Reply:
Oh, yeah no, absolutely, I can move them 👍
So just eg. `ABI.System` I assume?

Reply:
ABI.System works.  ABI.WinRT.Interop is probably fine too.

Reply:
Fixed in https://github.com/microsoft/CsWinRT/pull/1497/commits/bdf347023a7197212454b2756fead3b1323141bc.